### PR TITLE
modernized everything

### PR DESCRIPTION
**Pull request by cdaniluk**

### CODE REVIEW

File: .python-version [hunk 1]
```diff
@@ -0,0 +1 @@
+3.7.6/envs/ansible-role-netbox
```
Comment (sblack4) [inline .python-version:1]: I like this `version/envs/repo` for the pyenv env name. I think I'll steal it

**Review by dword4 — APPROVED** [review]

File: tasks/main.yml [hunk 2]
```diff
@@ -117,3 +102,27 @@
     group: "root"
     mode: 0400
   tags: ['netbox']
+
+# changed_when: False to stop idempotence checks from breaking
```
Comment (sblack4) [inline tasks/main.yml:106]: 🦆

**Review by sblack4 — APPROVED** [review]

**Comment by sblack4** [comment]
It looks like netbox doesn't come up for the GH Action container. Does it locally?

```
Failures/Skipped:

Port: tcp:8001: listening:
Expected
    <bool>: false
to equal
    <bool>: true
Port: tcp:8001: ip: skipped

Service: netbox: running:
Expected
    <bool>: false
to equal
    <bool>: true
```
from: https://github.com/rhythmictech/ansible-role-netbox/pull/1/checks?check_run_id=1150535240#step:4:672

**Comment by dgoodellrhy** [comment]
I'm going to merge this, we can handle Molecule Testing issues later

**Pull request merged**